ansys/dpf/core/server.py

```diff
@@ -221,15 +221,22 @@ def start_local_server(
 
     server = None
     n_attempts = 10
+    timeout = 10
+    timed_out = False
     for _ in range(n_attempts):
         try:
             server = DpfServer(
-                ansys_path, ip, port, as_global=as_global,
+                ansys_path, ip, port, timeout=timeout, as_global=as_global,
                 load_operators=load_operators, docker_name=docker_name
             )
             break
         except errors.InvalidPortError:  # allow socket in use errors
             port += 1
+        except TimeoutError:
+            if timed_out:
+                break
+            timeout *= 2.
```

Comment:
After a just a few attempts this timeout would grow quite large. Perhaps consider:
```suggestion
            timeout += 10.
```

Reply:
@akaszynski in the current implementation, the time out is only increased once

Reply:
If it fails for something other than a TimeoutError, it does not go through timeout *= 2, and it can only go through it once. The thing is I did not want to make an assumption on the absolute timeout value given in input (once timeout comes from the calling function) so doubling made more sense than adding *ten something*.